packages/datadog-plugin-pg/test/index.spec.js

```diff
@@ -484,10 +499,11 @@ describe('Plugin', () => {
 
         it('query config objects should be handled', done => {
           let queryText = ''
+
           const query = {
-            name: 'pgSelectQuery',
```

Comment:
This was actually an example of the bug happening. The previous test had a name and was therefore a prepared statement. The test asserts that the traceparent is present in the comment but it should actually not be present.